charts/portal/values.yaml

```
@@ -618,13 +622,18 @@ backend:
         cpu: 75m
         memory: 350M
     seeding:
-      testDataEnvironments: ""
+      # -- Enables dynamic seeding of bpn (dim.bpn) and did document location address (dim.didDocumentLocationAddress) of the operator;
+      # If set to `true` the data configured in the config map 'configmap-seeding-initialdata.yaml' will be taken to insert the initial data;
+      useInitial: false
```

should default to true
```suggestion
      useInitial: true
```

---

adjusted